code/controllers/voting.dm

```diff
@@ -20,9 +20,13 @@ var/global/list/round_voters = list() //Keeps track of the individuals voting fo
 			qdel(vote)
 		vote = src
 	spawn(0)
-		while(TRUE)
-			process()
-			sleep(10)
+		while(!gcDestroyed)
+			try

```

Comment:
The point of try/catch is that you don't need extra code to handle detecting and recovering from runtimes, which makes the code cleaner. Losing the stack trace is the only downside, but most of the time, it's simple enough to just manually trace back through the calls. Especially in a situation like this, with very few calls out of the vote controller code.